ext/devcontainer/devcontainer-feature.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "azure-dev-cli",
+    "version": "1.0.0",
+    "name": "Azure Dev CLI",
+    "documentationURL": "https://github.com/azure/azure-dev/tree/main/ext/devcontainer",
+    "description": "Installs the Azure Developer CLI along with needed dependencies.",
+    "options": {
+      "version": {
+        "type": "string",
+        "proposals": [
+          "latest"
+        ],
+        "default": "latest",
+        "description": "Select or enter an Azure Developer CLI version. (Available versions may vary by Linux distribution.)"
+      }
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+              "ms-azuretools.azure-dev",
+              "ms-azuretools.vscode-bicep"
```

Comment:
We may want to include more by default here.  @karolz-ms or @philliphoff thoughts?

Reply:
```suggestion
              "ms-azuretools.azure-dev",
              "ms-azuretools.vscode-bicep",
              "ms-vscode.vscode-node-azure-pack"
```